package/README.md

```diff
@@ -60,6 +20,28 @@ This project uses [Vite](https://vitejs.dev/) to build the bundle, using Vite's
 
 Some parts of the project are in TypeScript `.ts/.tsx` and other parts are in JavaScript `.js/.jsx`. TypeScript is used where its features are particularly beneficial, mostly in the models that describe forms' behavior.
 
+There are two entry points.
+
+One exports the forms as React components:
+
+```js
+import Userfront, { LoginForm } from "@userfront/toolkit/react";
+```
+
+The other registers the forms as Web Components, suitable for usage with most front-end frameworks or in plain JS:
+
+```js
+import Userfront from "@userfront/toolkit/web-components";
+```
+
+After import, the Web Components are registered and can be used anywhere in the app:
+
+```html
+<body>
+  <login-form tenant-id="mytenantid"></login-form>
```

Comment:
Is there a list anywhere of what components available to use? Would that be helpful to include somewhere in the README?

Reply:
good call. I added a components & props list